src/main/java/com/icthh/xm/tmf/ms/communication/service/SmppService.java

```diff
@@ -0,0 +1,71 @@
+package com.icthh.xm.tmf.ms.communication.service;
+
+import com.icthh.xm.tmf.ms.communication.config.SmppConfiguration;
+import lombok.extern.slf4j.Slf4j;
+import org.jsmpp.InvalidResponseException;
+import org.jsmpp.PDUException;
+import org.jsmpp.bean.*;
+import org.jsmpp.extra.NegativeResponseException;
+import org.jsmpp.extra.ResponseTimeoutException;
+import org.jsmpp.session.SMPPSession;
+import org.jsmpp.util.AbsoluteTimeFormatter;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.Date;
+
+@Slf4j
+@Component
+public class SmppService {
+
+
+    private final SmppConfiguration smppConfiguration;
+    private final AbsoluteTimeFormatter timeFormatter;
+
+    public SmppService(SmppConfiguration smppConfiguration) {
+        this.smppConfiguration = smppConfiguration;
+        this.timeFormatter = new AbsoluteTimeFormatter();
+    }
+
+
+    public void send(String destAdrrs, String message) {
+        SMPPSession session = null;
+        try {
+            session = this.smppConfiguration.getSession();
+            String messageId = session.submitShortMessage(
+                "CMT",
```

Comment:
What is CMT? And other hardcoded value? Let's move all of them in configuration.